Big O notation and python built-in data structures.py

```diff
@@ -0,0 +1,48 @@
+from typing import List, Tuple
+
+# We assume that all lists passed to functions are same length N
+
+
+# answers
+# 1 - n
+# 2 - 1
+# 3 - n^2
+# 4 - n
+# 5 - n^2
+# 6 - log n
+
+
+# O(N^2)
+def question1(first_list: List[int], second_list: List[int]) -> List[int]:
```

Comment:
Keep in mind, when we have multiple input parameters, BigO is represented like this: O(n*m)